package/com.unity.formats.usd/Runtime/Unity.Formats.USD.Runtime.asmdef

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Unity.Formats.USD.Runtime",
+    "references": [
+        "Unity.Timeline"
+    ],
+    "optionalUnityReferences": [],
+    "includePlatforms": [
+        "Editor",
+        "macOSStandalone",
```

Comment:
Logged an issue for the failing build : https://unity3d.atlassian.net/browse/FTV-75